### PR TITLE
Turkish translation metadata update

### PR DESCRIPTION
**Pull request by eersoy93**
- Changing my email address.
- Taking spam precaution for Arda Tanrıkulu's and my email addresses
- [TRANSLATIONS] media/doc/For_Turkish_Translation.txt changes.

  I won't translate ReactOS to pure Turkish but I may translate ReactOS
  to normal Turkish.

  This is my first ReactOS commit. \o/

## Purpose

I won't translate ReactOS to pure Turkish anymore.

JIRA issue: no JIRA issue

## Proposed changes

See my commit. My commit has proposed changes with unordered list.

## TODO

No TODOs.

### CODE REVIEW

File: media/doc/For_Turkish_Translation.txt [hunk 1]
```diff
@@ -1,7 +1,3 @@
-The ReactOS is translating to pure Turkish with new terminology. For new terminology, see: http://eersoy93.blogspot.com.tr/p/sozluk.html (in Turkish)
+ReactOS isn't translated to pure Turkish anymore by me but I may translate ReactOS to normal Turkish. For pure Turkish words, prefixes, suffixes etc. see for my pure Turkish dictionary: https://github.com/eersoy93/sozluklerim/blob/master/%C3%96z_T%C3%BCrk%C3%A7e_S%C3%B6zl%C3%BCk.md
```
Comment (AmineKhaldi) [inline media/doc/For_Turkish_Translation.txt]: This change seems to go beyond updating your email address, and I do not understand it.

**Comment by encodedpr** [comment]
A quick glance at the Turkish language wikipedia article reveals there was a Turkish Language Association (TDK) established in the 1930s whose mission was to replace the use of loan words from other languages with turkish rooted words. I think this might explain some of this "pure Turkish" thing.

File: media/doc/For_Turkish_Translation.txt (hunk 1 above)
Comment (eersoy93) [reply media/doc/For_Turkish_Translation.txt]: My commit has group of changes by me and my original commit's header is: Make some changes

**Comment by eersoy93** [comment]
NOTE: I'm very busy for translating ReactOS to Turkish.

File: media/doc/For_Turkish_Translation.txt (hunk 1 above)
Comment (eersoy93) [reply media/doc/For_Turkish_Translation.txt]: @AmineKhaldi Should I separate the commit to two commits?

Comment (yagoulas) [reply media/doc/For_Turkish_Translation.txt]: We generaly don't like huge comits that span the whole codebase.

**Comment by eersoy93** [comment]
I'm going to open new pull request.

**Pull request closed without merging**